### ISSUE TITLE
IJCAI 2023

### ISSUE THREAD

**Issue by inoue0426** [inoue0426/Paper-reading#11]
# IJCAI 2023 https://www.ijcai.org/proceedings/2023/

- CSGCL: Community-Strength-Enhanced Graph Contrastive Learning
  - https://www.ijcai.org/proceedings/2023/0229.pdf
  - https://github.com/HanChen-HUST/CSGCL
- Gapformer: Graph Transformer with Graph Pooling for Node Classification
  - https://www.ijcai.org/proceedings/2023/0244.pdf
  - Graph Transformer with Pooling
- Deep Multi-View Subspace Clustering with Anchor Graph
  - https://www.ijcai.org/proceedings/2023/0398.pdf
  - 2AE + spectral clustering. Looks suitable for DTI.
- SemiGNN-PPI: Self-Ensembling Multi-Graph Neural Network for Efficient and Generalizable Protein-Protein Interaction Prediction
  - https://www.ijcai.org/proceedings/2023/0554.pdf
  - Kind of contrastive learning by PPI and Label graph
- Toward Convex Manifolds: A Geometric Perspective for Deep Graph Clustering of Single-cell RNA-seq Data
  - Cell clustering by gcn encoder and decoder model
  - https://www.ijcai.org/proceedings/2023/0540.pdf
  - https://github.com/MMAMAR/scTConvexMan
- Multi-view Contrastive Learning Hypergraph Neural Network for Drug-Microbe-Disease Association Prediction
  - https://www.ijcai.org/proceedings/2023/0537.pdf
  - Drug-Microbe-Disease association prediction from smiles, microbe taxonomy, disease mesh. Contrastive learning with GCN, FCN, and MLP.
  - https://github.com/Liuluotao/MCHNN
- MolHF: A Hierarchical Normalizing Flow for Molecular Graph Generation
  - https://www.ijcai.org/proceedings/2023/0556.pdf
  - Simple normalizing flow.
  - https://github.com/violet-sto/MolHF
- A Systematic Survey of Chemical Pre-trained Models
  - https://www.ijcai.org/proceedings/2023/0760.pdf
  - Overview of chemical pre-trained model.
  - https://github.com/junxia97/awesome-pretrain-on-molecules
